class/Utils/Settings/Group.php

```diff
@@ -64,8 +71,9 @@ class Group {
 	 * @param string $name    Group name.
 	 * @param string $slug    Group slug.
 	 * @param string $section Section slug.
+	 * @param bool   $collapsed Group collapsed.
 	 */
-	public function __construct( $handle, $name, $slug, $section ) {
+	public function __construct( $handle, $name, $slug, $section, $collapsed = false ) {
```

Comment:
Better not use the param here, because if the section is collapsed or not is not really important. Just stick with the `collapsed` method, or even better, create the `collapse()` alias.

Then do something link this:

`$general->add_group( __( 'Content', 'notification' ), 'content' )->collapse()`

or

`$general->add_group( __( 'Content', 'notification' ), 'content' )->collapsed( true )`